.github/workflows/release-apt-get.yml

```diff
@@ -0,0 +1,52 @@
+name: "release-apt-get"
+on:
+  release:
+    types: [released]
+
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release Tag'
+        required: true
+        default: 'latest'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - uses: actions/checkout@v2
+
+    - uses: azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+    - name: 'Download Repo Client'
+      env:
+        AZ_SUB: ${{ secrets.AZURE_SUBSCRIPTION }}
+      run: |
+        az storage blob download --subscription  "$AZ_SUB" --account-name gitcitoolstore -c tools -n azure-repoapi-client_2.0.1_amd64.deb -f repoclient.deb --auth-mode login
```

Comment:
Is there no "more official" place to get this Debian package? I worry about hard-coding the version number "2.0.1" here...

Reply:
There is a way to get it from http://tux-devrepo.corp.microsoft.com, but I'm going to follow up with Kyle. I'm sure he likely had a good reason for not going that route when he set this up initially.

Reply:
I just don't want to be stuck with a stale version and only realizing when something hits the fan that we simply forgot to upgrade ;-)

Reply:
Ok, so the issue is that the tool is stored on http://tux-devrepo.corp.microsoft.com, which is inaccessible to machines that are not connected to corpnet. Kyle recommends setting up an internal pipeline to download latest and publish to blob storage. I have opened [this issue](https://github.com/github/git-client/issues/467) accordingly.